src/Optimizely/Optimizely.php

```diff
@@ -388,4 +369,170 @@ public function getForcedVariation($experimentKey, $userId)
             return null;
         }
     }
+
+
+    public function isFeatureEnabled($featureFlagKey, $userId, $attributes = null){
+
+        if (!$this->_isValid) {
+            $this->_logger->log(Logger::ERROR, "Datafile has invalid format. Failing '".__FUNCTION__."'");
+            return null;
+        }
+
+        if(!$featureFlagKey){
+            $this->_logger->log(Logger::ERROR, "Feature Flag key cannot be empty");
+            return null;
+        }
+
+        if(!$userId){
+            $this->_logger->log(Logger::ERROR, "User ID cannot be empty");
+            return null;
+        }
+
+        $feature_flag = $this->_config->getFeatureFlagFromKey($featureFlagKey);
+        if($feature_flag == new FeatureFlag){
+            // Error logged in ProjectConfig - getFeatureFlagFromKey
+            return null;
+        }
+
+        $decision = $this->_decisionService->getVariationForFeature($feature_flag, $userId, $attributes);
+        if(!$decision){
+            $this->_logger->log(Logger::INFO,"Feature Flag '{$featureFlagKey}' is not enabled for user '{$userId}'.");
+            return false;
+        }
+
+        if($decision["experiment"]){
```

Comment:
whu isset not used? btw it should be replaced by decision obj

Reply:
isset only checks that the dictionary has key defined the value is not null. 
When you provide a value in (), it should evaluate to true. 
"", 0, false, 0.0, null all evaluate to false. 
This will get replaced once the impl is finalized